### PR TITLE
Add breadcrumbs to pages in create flow

### LINKED ISSUE #1852: Can't escape from create flow
While I generally approve of simple UI, the create flow is perhaps too simple and strands the user without an expected "back" / "out" / "return" flow.

screenshot 2015-04-21 21 04 51

Help me it says... I'm lost and lonely!

### PR DESCRIPTION
**Pull request by spadgett**
Fixes #1852 

openshift_web_console

openshift_web_console

openshift_web_console

openshift_web_console

openshift_web_console

### CODE REVIEW

**Comment by spadgett** [comment]
@liggitt PTAL

File: assets/app/scripts/filters/resources.js [hunk 1]
```diff
@@ -380,4 +380,13 @@ angular.module('openshiftConsole')
 
       return false;
     };
+  })
+  .filter('projectOverviewURL', function(Navigate) {
```
Comment (liggitt) [inline assets/app/scripts/filters/resources.js]: this should take a projectName (which we'll always have from url params)

File: assets/app/views/catalog/images.html [hunk 2]
```diff
@@ -1,13 +1,13 @@
 <div ng-controller="ProjectController">
   <div ng-controller="CatalogImagesController">
     <div class="container">
+      <ol class="breadcrumb">
+        <li><a href="{{project | projectOverviewURL}}">{{project | displayName}}</a></li>
```
Comment (liggitt) [inline assets/app/views/catalog/images.html]: `{{projectName | projectOverviewURL}}`, otherwise the link is invalid until the project loads

Comment (spadgett) [inline assets/app/views/catalog/images.html]: Will fix.

File: assets/app/scripts/filters/resources.js (hunk 1 above)
Comment (spadgett) [inline assets/app/scripts/filters/resources.js]: Will fix.

Comment (liggitt) [inline assets/app/scripts/filters/resources.js]: I'm also fine with it taking a project or project name (if `p.metadata && p.metadata.name` exists, use that, etc)

File: assets/app/views/catalog/images.html (hunk 2 above)
Comment (liggitt) [inline assets/app/views/catalog/images.html]: or just inline `"project/{{projectName}}/overview"`

**Comment by spadgett** [comment]
@liggitt have another look. Thanks!

**Comment by liggitt** [comment]
[merge]

**Pull request merged**